tests/common.go

```diff
@@ -4064,12 +4064,41 @@ func ParseFlags() {
 			}
 		}
 
+		/*
+			Get TestSetID based on below precedence
+			1. Check if user has passed in command line and use
+			2. Check if user has set it has an env variable and use
+			3. Check if build.properties available with TestSetID
+			4. If not present create a new one
+		*/
 		val, ok := os.LookupEnv("DASH_UID")
-		if ok {
+		if testsetID != 0 {
+			dash.TestSetID = testsetID
```

Comment:
Since this check is also happening at line 4073, you can remove it at one place

Reply:
Have already removed this, its just one time now and if testsetID is not zero we set it as env variable